### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by vyaspranjal33**
added Link to LICENSE

NOTE: Please read the CONTRIBUTING.md guidelines before submitting your patch,
and ensure you followed them all:

https://github.com/Kong/kong/blob/master/CONTRIBUTING.md#contributing

### Summary

SUMMARY_GOES_HERE

### Full changelog

* [Implement ...]
* [Add related tests]
* ...

### Issues resolved

Fix #XXX

### CODE REVIEW

**Comment by vyaspranjal33** [comment]
@thibaultcha

**Comment by thibaultcha** [comment]
Please allow for some time for your PR to be reviewed; especially if it was opened during the weekend. Thanks.

File: README.md [hunk 1]
```diff
@@ -245,7 +245,7 @@ You may obtain a copy of the License at
 Unless required by applicable law or agreed to in writing, software
 distributed under the License is distributed on an "AS IS" BASIS,
 WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-See the License for the specific language governing permissions and
+See the [License](https://github.com/Kong/kong/blob/master/LICENSE) for the specific language governing permissions and
```
Comment (thibaultcha) [inline README.md]: A link here will not render (see https://github.com/Kong/kong/blob/e427c687944f809561825c0a20e14c7733d8be68/README.md). Also, we don't _have_ to have a link here. Lastly, the formatting broke and we are exceeding the 80 columns per line limit.

**Comment by thibaultcha** [comment]
Thank you, but it seems like this PR is unaligned with our requirements and doesn't provide any significant benefits.

**Pull request closed without merging**